pwndbg/heap/__init__.py

```diff
@@ -1,12 +1,81 @@
+import pwndbg.color.message as message
+import pwndbg.config
 import pwndbg.heap.heap
 import pwndbg.symbol
 
 current = None
 
 heap_chain_limit = pwndbg.config.Parameter('heap-dereference-limit', 8, 'number of bins to dereference')
 
+resolve_via_heuristic = pwndbg.config.Parameter('resolve-via-heuristic', False,
+                                                'Resolve some missing symbols via heuristics')
+
+main_arena = pwndbg.config.Parameter('main_arena', "0", 'main_arena address for heuristics')
```

Comment:
Hmm... those kinda bloats the config parameters a bit. Maybe we should do sth like "heapconfig" namespace and put all heap related params there?

I am fine with leaving them in config for now if u don't want to deal with it, we just have to make it clear what they mean/are for in their description --- I'd add text that they are related to heap or sth

Reply:
OK, I'll try to fix it!

Reply:
I added a new command called `heap_config` to show configs in `heap` scope(https://github.com/pwndbg/pwndbg/pull/1029/commits/289010da1dd4839bdb628b1f83390bbc352f88d5), does this fix the issue?